### ISSUE TITLE
Change order of transactions

### ISSUE THREAD

**Issue by Braqzen** [FuelLabs/block-explorer-v2#92]
# Summary

When a Tx is made it gets added to the Tx section at the bottom of the screen.
The problem is the order that it is added in is incorrect. 
It currently leaves the oldest Tx at the top and the latest Tx at the bottom so if you want to see your latest Tx's you must scroll all the way down (and potentially switch via pagination if enough Tx's happen).

Change the order to have the latest Tx be at the top.

# Screenshot
![1](https://user-images.githubusercontent.com/103777923/175615327-3128f324-f832-4251-8526-426a0c17d49b.png)

**Comment by LuizAsFight** [comment]
Blocked by #115

**Comment by LuizAsFight** [comment]
@Braqzen I was about to close this issue but realized that reverse pagination is not supported for this case:


```
query  {
  transactionsByOwner(last: 5, owner: "0x3e171209099ce85a66e38c2675395017de6da3bb8e55a931486f6333f605ff29") {
    edges {
      node {
        id
      }
    }
  }
}
```

then I get this error:
"reverse pagination isn't supported for this resource"

**Comment by LuizAsFight** [comment]
I will fix the ordering in front-end, and increase transactions by page to 20... it will help, but it's not a definitive fix for this issue

**Comment by LuizAsFight** [comment]
> @Braqzen I was about to close this issue but realized that reverse pagination is not supported for this case:
> 
> ```
> query  {
>   transactionsByOwner(last: 5, owner: "0x3e171209099ce85a66e38c2675395017de6da3bb8e55a931486f6333f605ff29") {
>     edges {
>       node {
>         id
>       }
>     }
>   }
> }
> ```
> 
> then I get this error: "reverse pagination isn't supported for this resource"

@luizstacio you know why this query doesn't support reverse pagination? is it possible to add reverse pagination to it?

**Comment by luizstacio** [comment]
> > @Braqzen I was about to close this issue but realized that reverse pagination is not supported for this case:
> > ```
> > query  {
> >   transactionsByOwner(last: 5, owner: "0x3e171209099ce85a66e38c2675395017de6da3bb8e55a931486f6333f605ff29") {
> >     edges {
> >       node {
> >         id
> >       }
> >     }
> >   }
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > then I get this error: "reverse pagination isn't supported for this resource"
> 
> @luizstacio you know why this query doesn't support reverse pagination? is it possible to add reverse pagination to it?

It looks like the fuel-core has an issue on this endpoint.

**Comment by luizstacio** [comment]
@LuizAsFight I was looking deep into the code. I found that this is not supported because of a hard limitation on the storage. It should be possible to do this once we move the block explorer to use the indexer. For now, can we add pagination on this screen? Maybe create a new issue for that. And keep this open.

**Comment by LuizAsFight** [comment]
@luizstacio there is already an issue #93  for that but it's P3, that's why I didn't do it yet as I'm working on P2s first
